packages/kbn-reporting/public/share/share_context_menu/register_pdf_png_modal_reporting.tsx

```diff
@@ -322,6 +320,9 @@ export const reportingExportModalProvider = ({
             ),
             'data-test-subj': 'queueReportSuccess',
           });
+          if (onClose) {
+            onClose();
+          }
```

Comment:
Maybe we can delegate the responsibility of closing to some UI component much closer to where the sharing modal was opened, for instance in the export content component. what do you think?